provider/datagen/src/driver.rs

```diff
@@ -109,6 +112,16 @@ impl DatagenDriver {
         Self { fallback, ..self }
     }
 
+    /// Sets the base language handling in runtime fallback mode.
+    ///
+    /// For more details, see [`BaseLanguageHandling`].
+    pub fn with_base_language_handling(self, base_language_handling: BaseLanguageHandling) -> Self {
```

Comment:
issue: this option is only valid for a specific fallback mode. I'd prefer modelling it with a new fallback mode

Reply:
I think it applies for both `Runtime` and `RuntimeManual`, in addition to applying to `PreferredForExporter` in certain cases; an user who wants to enable resolvedLocale queries but doesn't want to pick a specific fallback type. This would mean we would need to add 3 more fallback variants + any other variant that is added in the future that is similar to `Runtime`.

Reply:
Good point about `PreferredForExporter`. With that option, the exporter can state its preferred fallback mode, but that question is completely separate from whether to include the base languages.

Reply:
An exporter does not state its preferred fallback mode, it states `supports_built_in_fallback`. Datagen translates this to an actual mode, which we are free to change: `true` can be `RuntimeThin`, and `false` `HybridThin`, as you are currently making this the default already.